doc/api/process.md

```diff
@@ -3005,6 +3005,10 @@ console.log(resourceUsage());
 */
 ```
 
+## `process.runNextTicks()`
+
+Ensure all tasks in the V8 micro-task queue have completed. This will cause any callback passed to [`process.nextTick()`][], [`queueMicrotasks()`][] or to [`.then()`][`promise.then()`] or [`.catch`][`promise.catch()`] of an immediately resolving or rejecting promise to be executed.
```

Comment:
```suggestion
Ensures all tasks in the V8 micro-task queue have completed. This will cause any callback passed to [`process.nextTick()`][], [`queueMicrotasks()`][] or to [`.then()`][`promise.then()`] or [`.catch`][`promise.catch()`] of an immediately resolving or rejecting promise to be executed.
```